.github/workflows/preview.yaml

```
@@ -62,27 +62,34 @@ jobs:
       run: |
         npm run build
 
-    - name: Build and push server image
-      uses: docker/build-push-action@v1
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Login to DockerHub
+      uses: docker/login-action@v1
       with:
-        path: server
         registry: ${{ env.DOCKER_REGISTRY }}
         username: _json_key
         password: ${{ secrets.GKE_SA_KEY }}
+
+    - name: Build and push server image v2
+      uses: docker/build-push-action@v2
+      with:
+        context: server
+        pull: true
         push: true
-        repository: ${{ secrets.GKE_PROJECT }}/webui/nuxeo-web-ui/server
-        tags: ${{ env.VERSION }}
+        file: server/Dockerfile
+        tags: ${{ env.DOCKER_REGISTRY }}/${{ secrets.GKE_PROJECT }}/webui/nuxeo-web-ui/server:${{ env.VERSION }}
```

Isn't this the same as?
```suggestion
        tags: ${{ env.DOCKER_REGISTRY }}/${{ env. DOCKER_IMAGE }}/server:${{ env.VERSION }}
```

---

It is in fact. I'll update this one too. Thanks